http/userTakenAssessment.ts

```diff
@@ -72,7 +73,7 @@ export const fetchUserTakenAssessment = async (token: string, id: any) => {
     if (!res.data) {
       return;
     }
-    console.log('responce', res.data.data);
+    console.log('response', res.data.data);
```

Comment:
Remove console.log

Reply:
Done! and i did some more optimizations and some little SEO

Reply:
@billmal071

Reply:
My teammate just pushed and there's a lot of conflicts with the file, I'm resolving them